packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/util.ts

```diff
@@ -90,3 +91,11 @@ export function mapTagMapToCxschema(tagMap: Record<string, string>): cxschema.Ta
   return Object.entries(tagMap)
     .map(([key, value]) => ({ key, value }));
 }
+
+/**
+ * Return the targetGroupName from targetGroupArn
+ */
+export function targetGroupNameFromArn(targetGroupArn: string) {
+  const arnParts = Fn.split('/', targetGroupArn);
```

Comment:
We have a function in the `core` Stack for this already: `Stack.splitArn()`. See how it's used [here](https://github.com/aws/aws-cdk/blob/203c42b792e46536eaf21fa48277985c3409b65c/packages/%40aws-cdk/aws-kinesisanalytics-flink/lib/application.ts#L230), for example.